src/exampleTracks.js

```diff
@@ -0,0 +1,2878 @@
+export const exampleTracks = [
```

Comment:
Node can import JSON. Just put this JSON in a `exampleTracks.json` and `import exampleTracks from "./exampletTracks.json"`.